includes/import/abstract-wc-product-importer.php

```diff
@@ -211,6 +211,7 @@ protected function get_product_object( $data ) {
 	protected function process_item( $data ) {
 		try {
 			do_action( 'woocommerce_product_import_before_process_item', $data );
+			$data = apply_filters( 'woocommerce_product_import_before_process_item', $data );
```

Comment:
This need to have a different name for this filter, also the name for this filter doesn't make sense, this name using "before" sounds good only for actions.

Reply:
Of couse, I thought I'd let the filter name be up to you guys so I just copy pasted the action name for the sake of having the filter there. Any suggestions? Something like `woocommerce_product_import_process_item_data`?